app/javascript/components/gtl/DataTable.jsx

```diff
@@ -60,13 +67,16 @@ export const DataTable = ({
       <tr>
         {!inEditMode() && !noCheckboxes() &&
         <th className="narrow table-view-pf-select">
+          <label className="hiddenCheckboxLabel" id="selectAll" aria-hidden="true">selectAll</label>
```

Comment:
Is the text here correct? I think it should be translated and written as proper text...

Reply:
I did not feel the need to translate it as this is a hidden text.

Reply:
Well, isn't the purpose of the label to hint the screenreader? In that case even if the label is not displayed, the screen reader will be linking to it due to the `aria-labelledby` on the checkbox. So in that case I think it should be a proper text and translated...